docs/github/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+<img src="sm-grad-logo.png" >
+
+# Contributing
+
+## Getting Started
+
+This repo uses [Husky](https://typicode.github.io/husky/#/) with [Commitlint](https://commitlint.js.org/#/) to format commit messages. To install, run:
+
+```sh
+yarn prepare
+```
+
+All commit messages should follow Commitlint's conventions, such as:
+
+```
+feat: move request detail admin tools and add twitter-specific image option
+```
+
+## Branch Conventions
+
+The production branch for this repository is `main`, and `development` is for staging. All other branches should follow the following conventions:
+
+- `chore/name-of-branch`: refactoring or updating existing code
+- `docs/name-of-branch`: adding documentation
+- `feat/name-of-branch`: creating a feature
+- `fix/name-of-branch`: fixing a bug
+
+## Making Pull Requests
+
+All pull requests apart from deployment should be made from the feature branch to `main`.
+
+Before making a pull request, please make sure to rebase your branch with `main` to ensure all previous features are included.
```

Comment:
It should be clear that rebase should be from origin/main, and that you need to fetch before you rebase

Reply:
Gotcha! Adding it now.